src/layout/header/Header.tsx

```diff
@@ -1,15 +1,30 @@
 import { Layout } from 'antd'
 import { MenuOutlined } from '@ant-design/icons'
 import { useContext } from 'react'
-import { LayoutContextInterface, LayoutCtx } from 'src/layout/LayoutContext'
-
+import { LayoutContextInterface, LayoutCtx } from '../LayoutContext'
+import { useTheme } from '../ThemeContext'
+import { BulbFilled, BulbOutlined } from '@ant-design/icons'
 const { Header } = Layout
 
 const MainHeader = () => {
   const { setDrawerOpen } = useContext<LayoutContextInterface>(LayoutCtx)
+  const { isDarkTheme, toggleTheme } = useTheme()
   return (
-    <Header style={{ background: 'white' }} className="hideOnDesktop">
-      <MenuOutlined onClick={() => setDrawerOpen(true)} />
+    <Header
+      style={{
+        background: isDarkTheme ? '#141414' : '#fff',
+        display: 'flex',
+        justifyContent: 'flex-end',
+        alignItems: 'center',
+      }}
+      className="_hideOnDesktop">
```

Comment:
adding
```suggestion
      className={cn("_hideOnDesktop", "header", { dark: isDarkTheme })}>
```
would make the structure and styling more separated